src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/SpecimenDetails.java

```diff
@@ -53,11 +52,6 @@
     private final SpecimenCharacteristicType characteristicType =
         SpecimenCharacteristicType.fromString(extractSegment(SpecimenCharacteristicType.KEY_QUALIFIER));
 
-    @Getter(lazy = true)
```

Comment:
is `May optionally also be present qualified with {@code +FS}.` still relevant in the class javadoc?